daemon/daemon.go

```diff
@@ -967,3 +968,35 @@ func getDefaultRouteMtu() (int, error) {
 	}
 	return 0, errNoDefaultRoute
 }
+
+// verifyContainerSettings performs validation of the hostconfig and config
+// structures.
+func (daemon *Daemon) verifyContainerSettings(hostConfig *runconfig.HostConfig, config *runconfig.Config) ([]string, error) {
+
+	// First perform verification of settings common across all platforms.
```

Comment:
Ah, another 3-line comment :) I'd probably say this is OK. In the context of the code as it stands now, I would say it's obvious the check is being done on the daemon rather than the client. I added that comment way back when I did the original move from client to server. It's truly redundant now and I probably should never have put it in in the first place